.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,3 +30,4 @@
 - [ ] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
 - [ ] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
 - [ ] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)
+- [ ] If a new feature was added, sufficient unit & integration test coverage is included
```

Comment:
It depends, do we have any definition of it internally? We can't say 100% coverage, because it doesn't necessarily make sense in all cases. Also difficult to say "no reduction in code coverage", as we don't have a good way to calculate this right now.

Reply:
How about something like this:
```suggestion
- [ ] All code changes have 100% unit and integration test coverage. If testing is not applicable or too difficult to justify doing, the reasoning should be documented explicitly in the PR.
```